yarn-completion.bash

```diff
@@ -40,8 +41,11 @@ __yarn_get_package_fields() {
 		cwd="${cwd%/*}"
 	done
 
-	while getopts ":gt:" opt; do
+	while getopts ":gp:t:" opt; do
```

Comment:
@dsifford I am not sure if I set the colons right or not. It works but I am struggling to understand under which circumstances you have to set a colon after the option.

`:` at the start of the getopts string -> every option is optional
`:` after a letter (`p:` and `t:`) -> this value must have an option argument

is this correct?

Reply:
Almost. If there is a leading colon before the start for the optstring, it enables silent error reporting. A decent explanation of that [can be found here](https://wiki.bash-hackers.org/howto/getopts_tutorial#error_reporting)